submissions/Roophee/JS - DOM/index.js

```diff
@@ -0,0 +1,111 @@
+"use strict";
+
+const clickBurger = document.querySelector(".burger");
+const clickContent = document.querySelector(".content");
+const clickHeader = document.querySelector(".header");
+const clickfooter = document.querySelector(".footer");
+
+const deBar = document.querySelector(".aside");
+
+
+// MOVE ASIDE & CONTENT
+
+function hideSideBar(e) { 
+    e.preventDefault();
+    deBar.classList.toggle('hide__sidebar');
+    clickContent.classList.toggle('move_content');
+    clickBurger.classList.toggle('burger__stop');
+    console.log(e.type)
+
+};
+
+clickBurger.addEventListener('click', hideSideBar);
+
+// Enter-key event show-hide aside
+
+clickBurger.addEventListener('keydown', function (e) {
+    e.preventDefault();
+    if (e.keyCode == "13") {
+        deBar.classList.toggle('hide__sidebar');
+        clickContent.classList.toggle('move_content');
+        clickBurger.classList.toggle('burger__stop');
+    }
+}
+);
+
+// CLOSE ASIDE WHEN MOUSE CLICK OUT
+
+function hideSideBarClickAway(e) {
+    e.preventDefault();
+    for (let i = 0; i < deBar.classList.length; i++) {
+        if (deBar.classList[i] === 'hide__sidebar') {
+            deBar.classList.toggle('hide__sidebar');
+            clickBurger.classList.toggle('burger__stop');
+            clickContent.classList.toggle('move_content');
+        };
+    };
+};
+
+clickContent.addEventListener('click', hideSideBarClickAway);
```

Comment:
In this case better to add one `eventListener` on `window` or `document` instead of three for closing navigation on click outside of navigation.